iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -83,6 +83,26 @@ def LinalgExt_ScatterOp : LinalgExt_Op<"scatter"> {
           .getShape()
           .back();
     }
+
+    Value updates() {
```

Comment:
Thanks for adding these. After I mentioned it, I realized I hadnt actually added this. It was part of my local changes. This looks good.